src/clj/runbld/notifications/slack.clj

```diff
@@ -0,0 +1,38 @@
+(ns runbld.notifications.slack
+  (:refer-clojure :exclude [send])
+  (:require [runbld.schema :refer :all]
+            [schema.core :as s]
+            [cheshire.core :as json]
+            [clj-http.client :as http]
+            [runbld.io :as io]
+            [runbld.notifications :as n]
+            [runbld.store :as store]
+            [stencil.core :as mustache]))
+
+(s/defn send :- s/Any
+  [opts :- MainOpts
+   ctx  :- NotifyCtx]
+  (let [f    (-> opts :slack :template)
+        tmpl (-> f io/resolve-resource slurp)
+        js   (mustache/render-string tmpl ctx)]
+    (http/post (-> opts :slack :hook)
+               {:body js})))
+
+(defn send?
+  "Send a slack alert depending on configs"
```

Comment:
This docstring doesn't match what the function does (just returns truthy or falsey)